assets/js/types/type-defs/addresses.ts

```diff
@@ -0,0 +1,13 @@
+export interface Address {
+	address_1: string;
+	address_2?: string;
+	city?: string;
+	company?: string;
+	country: string;
+	email: string;
+	first_name: string;
+	last_name: string;
+	phone?: string;
+	postcode: string;
+	state?: string;
+}
```

Comment:
Could we maybe use all/part of this: 

https://github.com/woocommerce/woocommerce-blocks/blob/5dac3a0e74e2de9c4a4bfecca3f15b380b05433f/assets/js/settings/shared/default-address-fields.ts#L40